tests-behat/bootstrap/Context.php

```diff
@@ -480,7 +469,7 @@ protected function jqueryWait($duration = 5000)
             }
         }
 
-        throw new \Exception('JQuery did not finished within a given time limit');
+        throw new \Exception('jQuery did not finished within a time limit');
```

Comment:
Same here: jQuery did not finish within the time limit

Reply:
this PR has that PR merged in, so will be resolved once merged into develop later itself...